catalyst-gateway/bin/src/service/common/objects/voter_info.rs

```diff
@@ -9,19 +9,22 @@ use super::{delegate_public_key::DelegatePublicKey, voter_group_id::VoterGroupId
 pub(crate) struct VoterInfo {
     /// Voter's voting power.
     /// This is the true voting power, subject to minimum voting power and max cap.
-    #[oai(validator(minimum(value = "0")))]
+    // TODO - Recheck max value
+    #[oai(validator(minimum(value = "0"), maximum(value = "1000000000000000000")))]
     voting_power: i64,
 
     /// Voter's voting group.
     voting_group: VoterGroupId,
 
     /// Total voting power delegated to this voter.
     /// This is not capped and not subject to minimum voting power.
-    #[oai(validator(minimum(value = "0")))]
+    // TODO - Recheck max value
+    #[oai(validator(minimum(value = "0"), maximum(value = "1000000000000000000")))]
     delegations_power: i64,
 
     /// Number of registration which delegated to this voter.
-    #[oai(validator(minimum(value = "0")))]
+    // TODO - Recheck max value
+    #[oai(validator(minimum(value = "0"), maximum(value = "1000")))]
```

Comment:
as maximum boundary not quit clear at this moment, lets assigned it to the max `i64` value as you did previously.